mongo/mongo.go

```diff
@@ -257,20 +259,32 @@ func (m *Mongo) checkoutConnection(server driver.Server) (conn driver.Connection
 }
 
 // see https://github.com/mongodb/mongo-go-driver/blob/v1.3.4/x/mongo/driver/operation.go#L532-L561
-func (m *Mongo) roundTrip(conn driver.Connection, req []byte) (res []byte, err error) {
+func (m *Mongo) roundTrip(conn driver.Connection, req []byte, unacknowledged bool) (res []byte, err error) {
 	defer func(start time.Time) {
 		addressTag := fmt.Sprintf("address:%s", conn.Address().String())
-		_ = m.statsd.Distribution("request_size", float64(len(req)), []string{addressTag}, 1)
-		if err == nil {
-			_ = m.statsd.Distribution("response_size", float64(len(res)), []string{addressTag}, 1)
+		tags := []string{addressTag}
+		if unacknowledged {
+			tags = append(tags, "unacknowledged:true")
```

Comment:
we should add the `unacknowledged` tag with both `true` and `false` values; would suggest just using a `fmt.Sprintf`. That way we can filter for both in metrics

Reply:
Changed in latest commit.